src/bundle/Resources/public/scss/_modals.scss

```diff
@@ -16,6 +16,7 @@
 
         .close {
             @include close-button();
+            padding: calculateRem(2px);
```

Comment:
for  dart-sass validness it should be either reversed with mixin, or done like that
```
@include close-button();

& {
    padding: calculateRem(2px);
}
```
as `close-button` uses nested selector which sass has problem with mixed declarations https://sass-lang.com/documentation/breaking-changes/mixed-decls/